lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -210,7 +214,7 @@ export default function ContentSelector({
               variant="primary"
               data-testid="blackboard-file-button"
             >
-              Select PDF from Blackboard
+              <FormattedMessage defaultMessage="Select PDF from Blackboard" />
```

Comment:
None of these examples show anything particularly interesting, but to give a flavor of other ways this can work:

```
const salutation = 'Dude';
<FormattedMessage 
  defaultMessage="Whatever, {salutation}. <a>Check out our website.</a>"
  values={
   salutation,
    a: chunks => <a href="http://www.cool.com">{chunks}</a>
  } />
```

Where `salutation` is dynamic and `<a>` allows formatting with rich text/JSX. Yes, you can nest components :).